code/modules/jobs/job_types/ostia.dm

```diff
@@ -0,0 +1,273 @@
+	supervisors = "You obey the governor, and the legion military, whom you are also loyal too."
+	display_order = JOB_DISPLAY_ORDER_F13OSTIASUBJECT 
+	exp_requirements = 0
+	outfit = /datum/outfit/job/CaesarsLegiontown/subject
+
+	loadout_options = list(
+		/datum/outfit/loadout/subjectoflegion,
+		/datum/outfit/loadout/legionshopkeeper,
+		/datum/outfit/loadout/legiongladiator,
+		/datum/outfit/loadout/legionbarkeep,
+		)
+
+	matchmaking_allowed = list(
+		/datum/matchmaking_pref/friend = list(
+			/datum/job/CaesarsLegiontown/slave,
+		),
+		/datum/matchmaking_pref/rival = list(
+			/datum/job/CaesarsLegiontown/slave,
+		),
+	)
+
+	access = list(ACCESS_BAR, ACCESS_TOWN)
+	minimal_access = list(ACCESS_BAR, ACCESS_TOWN)
+
+/datum/outfit/job/CaesarsLegiontown/subject/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	ADD_TRAIT(H, TRAIT_MARS_TEACH, src)
+
+/datum/outfit/job/CaesarsLegiontown/subject
+	name = "Legion Subject"
+	jobtype = /datum/outfit/job/CaesarsLegiontown/subject
+	id = /obj/item/card/id/dogtag/legimmune
+	uniform = /obj/item/clothing/under/civ/roman_centurion
+	shoes =	/obj/item/clothing/shoes/roman
+	ears = /obj/item/radio
+
+
+/datum/outfit/loadout/subjectoflegion
+	name = "Legion Subject"
+	backpack_contents = list(		
+		/obj/item/ammo_box/a357 = 1,
+		/obj/item/gun/ballistic/revolver/colt357 = 1,
+		/obj/item/stack/f13Cash/random/denarius/low = 2,
+		/obj/item/flashlight/lantern = 1,
+	)
+
+/datum/outfit/loadout/legiongladiator
+	name = "Legion Gladiator"
+	backpack_contents = list(
+		/obj/item/clothing/head/helmet/gladiator = 1,
+		/obj/item/clothing/under/gladiator = 1,
+		/obj/item/clothing/shoes/roman = 1,
+		/obj/item/flashlight/lantern = 1,
+		/obj/item/melee/onehanded/machete = 1,
+		/obj/item/stack/f13Cash/random/denarius/low = 1,
+		/obj/item/book/granter/trait/bigleagues = 1,
+		/obj/item/shield/riot/buckler = 1,
+		/obj/item/lighter = 1,
+		)
+
+/datum/outfit/loadout/legionbarkeep
+	name = "Legion Restaurant"
+	backpack_contents = list(
+		/obj/item/clothing/head/f13/servant = 2,
+		/obj/item/clothing/under/civ/spanish_sailor = 2,
+		/obj/item/clothing/gloves/f13/crudemedical = 2,
+		/obj/item/clothing/shoes/roman = 2,
+		/obj/item/flashlight/lantern = 2,
+		/obj/item/reagent_containers/food/condiment/flour = 2,
+		/obj/item/storage/box/bowls = 2,
+		/obj/item/melee/onehanded/knife/cosmicdirty = 1,
+		/obj/item/soap/homemade = 1,
+		/obj/item/lighter = 1,
+		)
+
+/datum/job/CaesarsLegiontown/governor
+	title = "Legion governor of Ostia"
+	flag = F13LEGIONGOVERNOR
+	total_positions = 1
+	spawn_positions = 1
+	description = "The Civilian Leader of the Legion port of Ostia. You organise this city, make sure everything is in order, lead arrestation of troublemaker, organise slavesales with the slave master... You have only a limited power over the subject, and have no say in the military. You must have a latin name."
```

Comment:
organize, slave sales, capitalize Slave Master & Latin*
Suggest: "arrest troublemakers" instead of 'lead arrestation of troublemaker' & "You have limited authority over slaves" instead of 'You have only a limited power over the subject'